pkgs/by-name/ha/hatch/package.nix

```
@@ -69,19 +74,13 @@ python3.pkgs.buildPythonApplication rec {
     "test_editable_exact_force_include_option"
     "test_editable_exact_force_include_build_data_precedence"
     "test_editable_pth"
-    # AssertionError: assert len(extract_installed_requirements(output.splitlines())) > 0
-    "test_creation_allow_system_packages"
-    # cli table output mismatch
-    "test_context_formatting"
     # expects sh, finds bash
     "test_all"
     "test_already_installed_update_flag"
     "test_already_installed_update_prompt"
-    # unmet expectations about the binary module we provide
-    "test_dependency_not_found"
-    "test_marker_unmet"
-    # output capturing mismatch, likely stdout/stderr mixup
-    "test_no_compatibility_check_if_exists"
+    # Loosen hatchling runtime version dependency
+    "test_core"
+    "test_correct"
```

```suggestion
    "test_correct"
    "test_resolve"
```

not exactly sure why this fails on aarch64-linux but not x86_64